ibc-tester/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG SSC_VERSION=0.2.1
+ARG SAGAOS_VERSION=0.9.0
+
+FROM sagaxyz/sagaos:${SAGAOS_VERSION} AS sagaos
+FROM sagaxyz/ssc:${SSC_VERSION} AS ssc
+FROM ubuntu:24.04
```

Comment:
Do you need latest ubuntu? Maybe alpine would work and would be way lighter and easier to allocate.

Reply:
It's probably not necessary but when I tried alpine initially, I got a bunch of weird errors like `/usr/bin/sscd: cannot execute: required file not found` (same for sagaosd). It somewhat related to a different binary loader in Alpine and I couldn't fix it quickly enough, not even sure if it's possible to do without rebuilding both from the sources.